BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceViewController.swift

```diff
@@ -248,7 +248,8 @@ extension DigitalInvoiceViewController: UITableViewDelegate, UITableViewDataSour
                                                                            index: indexPath.row,
                                                                            invoiceNumTotal: invoice.numTotal,
                                                                            invoiceLineItemsCount:
-                                                                                invoice.lineItems.count)
+                                                                           invoice.lineItems.count,
+                                                                           nameMaxCharactersCount: 300)
```

Comment:
[Suggestion]: Could you please extract this in `Constants` extensions?
And let's limit it to 150

Reply:
Sure, done!